examples/3d/skybox.rs

```diff
@@ -157,7 +157,9 @@ fn asset_loaded(
         // NOTE: PNGs do not have any metadata that could indicate they contain a cubemap texture,
         // so they appear as one texture. The following code reconfigures the texture as necessary.
         if image.texture_descriptor.array_layer_count() == 1 {
-            image.reinterpret_stacked_2d_as_array(image.height() / image.width());
+            image
+                .reinterpret_stacked_2d_as_array(image.height() / image.width())
+                .unwrap();
```

Comment:
This is a user-facing example. This should, at the minimum, use expect, and in the best case handle the error/bubble it up.

Reply:
It behaves the same way as before. Thought about adding a comment. Good call on using expect - let me know what you think